### PR TITLE
www: deploy page remove fontawesome, jquery and dependencies, google fonts, and add a new iconfont

### PR DESCRIPTION
**Pull request by arnaudgs**
You can check the result of the page by following the instruction here :
https://github.com/arnogues/vates-stuff/tree/main/xcp-ng-www/cypress

If you want the real source of the icons check info here :
https://github.com/arnogues/vates-stuff/tree/main/xcp-ng-www/icon-font

### CODE REVIEW

File: src/xenserver/opt/xensource/www/XCP-ng-index.html [hunk 1]
```diff
@@ -1,297 +1,24 @@
...
     function copyToClipboard(element) {
-      var $temp = $('<input>');
-      $('body').append($temp);
-      $temp.val($(element).text()).select();
-      document.execCommand('copy');
-      $temp.remove();
+      element = typeof element === "string" ? document.querySelector(element) : element;
+      navigator.clipboard.writeText(element.textContent)
     }
   </script>
```
Comment (pdonias) [inline src/xenserver/opt/xensource/www/XCP-ng-index.html]: Is this used?

**Review by pdonias — CHANGES_REQUESTED** [review]

File: src/xenserver/opt/xensource/www/XCP-ng-index.html (hunk 1 above)
Comment (arnaudgs) [reply src/xenserver/opt/xensource/www/XCP-ng-index.html]: That's true, that I haven't seen any code that call this function
I remove it now

**Comment by stormi** [comment]
Looks good to me, with one comment: the first line of the commit message must not be larger than 70 chars, per our [commit message conventions](https://xcp-ng.org/docs/develprocess.html#commit-message-conventions). You can now squash your commits and update the message.

**Review by pdonias — APPROVED** [review]

**Comment by arnaudgs** [comment]
Squash done

**Pull request merged**